mempool/src/tests/node.rs

```diff
@@ -544,8 +544,14 @@ fn setup_node_network_interfaces(
 
     // Add Secondary network if the node has one
     if let Some(secondary_network_id) = node.secondary_network() {
+        if node.primary_network() == secondary_network_id {
```

Comment:
Do we need to check this in the tests?

Reply:
Not every time, but I wanted to make sure there were no lingering problems given it's an invariant we expect.  The tests were being too slow on my computer to run